packages/fastq_tools.py

```diff
@@ -0,0 +1,58 @@
+from typing import Union
+def count_gc(seq: str) -> float:
+    """
+    Counts gc content of sequence.
+    Arguments: 
+        -seq (str): dna/rna seqeunce.
+    Return: 
+        -float - percent of GC content in sequence.
+    """
+    gc = seq.lower().count('g') + seq.lower().count('c')
+    answer = gc / len(seq) * 100
+    return round(answer, 2)
+def count_qscore(score_string: str) -> float:
```

Comment:
```suggestion
    return round(answer, 2)
    
def count_qscore(score_string: str) -> float:
```

пустая строка должна разделять фунцкции